### ISSUE TITLE
[Question]:  how to abort requests to third-party domains

### ISSUE THREAD

**Issue by orennu** [microsoft/playwright#8116]
### Your question

hi ,
we see that new page keyword is pending all requests (including sub-sequent requests from 3rd party like google analytics) before we can interact with the DOM.
is this by design?
is it possible to change it via configuration or argument?

basically when we test our website, there are calls to all kinds of 3rd party providers which are not under our control and have no impact on the DOM so we don't need them to succeed, hence we don't want to wait for them before we can start our test.
due to random network or other issues, which are causing the response to take long time or timeout, we are getting timeout errors which cause the test to fail.

**Comment by aslushnikov** [comment]
Hey @orennu!

You can use [request interception](https://playwright.dev/docs/network#handle-requests) to abort all requests to the URLs you don't like.

For example:

```js
context.route('http://url.idontlike.com/**', request => request.abort());
```

**Comment by pavelfeldman** [comment]
Please let us know if you still need something in a new issue.
